pkg/metrics/metrics.go

```diff
@@ -535,111 +343,645 @@ var (
 	//
 	// Deprecated: This metric can be removed in 1.6 as
 	// KVStoreOperationsDuration provides the count along with duration
-	KVStoreOperationsTotal = prometheus.NewCounterVec(prometheus.CounterOpts{
```

Comment:
@aanm I had to convert this metric, which doesn't exist in 1.6.

Reply:
Looks good